### PR TITLE
Feat: OIDC change

### PR DESCRIPTION
**Pull request by ChrisRousey**

### CODE REVIEW

**Comment by flippingbits** [comment]
Is this PR a duplicate of #84?

**Comment by ChrisRousey** [comment]
> Is this PR a duplicate of #84?

No. i origninally opened this as a PR for the OIDC changes, but @HknLof rightfully mentioned i should make framework upgrades in a dedicated PR in case we need to rollback. #84 builds on this PR. It felt like we needed the updated framework for OIDC, because it had some weird behaviour in our older version

File: gradle.properties [hunk 1]
```diff
@@ -4,3 +4,4 @@ quarkusPlatformArtifactId=quarkus-bom
 quarkusPluginId=io.quarkus
 quarkusPlatformGroupId=io.quarkus.platform
 quarkusPlatformVersion=2.12.0.Final
+org.gradle.jvmargs=-Xmx4096m
```
Comment (flippingbits) [inline gradle.properties:7]: Is this related to the removal of the dependency?

Comment (ChrisRousey) [reply gradle.properties:7]: oops, forgot to not commit this config. it's a local config, to avoid the no java heap space error that happens a lot when working locally with the ee extension

File: platform-api/src/main/java/io/datacater/core/kubernetes/PythonRunnerPool.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import io.fabric8.kubernetes.api.model.Pod;
 import io.fabric8.kubernetes.api.model.apps.StatefulSet;
 import io.fabric8.kubernetes.client.KubernetesClient;
+import io.quarkus.runtime.Startup;
```
[the remark this answers is not in the document]
Comment (ChrisRousey) [reply platform-api/src/main/java/io/datacater/core/kubernetes/PythonRunnerPool.java:7]: yes 👍🏼

**Review by flippingbits — APPROVED** [review]
LGTM
